OWNERS

```diff
@@ -0,0 +1,6 @@
+approvers:
```

Comment:
This file can be removed, we do not use the kubernetes infrastructure/bots for Gluster projects.